src/py42/exceptions.py

```diff
@@ -96,7 +87,27 @@ def __init__(self, exception):
         super(Py42SessionInitializationError, self).__init__(exception, error_message)
 
 
-class Py42UserAlreadyAddedError(Py42HTTPError):
+class Py42BadRequestError(Py42HTTPError):
```

Comment:
These had to be moved up because super classes have to come before their inheritors if they are in the same file